main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"github.com/mr-Evgeny/go_final_project/config"
+	"github.com/mr-Evgeny/go_final_project/database"
+	"github.com/mr-Evgeny/go_final_project/handler"
+	"log"
+	"net/http"
+)
+
+func main() {
+	config.Init()
+	database.Connect()
+
+	fileServer := http.FileServer(http.Dir(config.SERVER_FILEDIR))
+	http.Handle("/", fileServer)
+	http.HandleFunc("/api/signin", handler.Sign)
```

Comment:
Тут можно использовать такую конструкцию, что не городить внутри проверки
```suggestion
	http.HandleFunc("POST /api/signin", handler.Sign)
```